### DISCUSSION TITLE
web ui password

### DISCUSSION THREAD

**Discussion by alex3dn** [wg-easy/wg-easy#607] [Q&A] [answered]
Please help. How to change web ui password ?

**Comment by eandersons** [comment, marked as answer]
Password is defined with the environment variable `PASSWORD` (see [Options](/wg-easy/wg-easy#options)). In order to change the password, the running container must be stopped and removed and then recreated with a new value of the environment variable `PASSWORD`.

In case you have forgotten password and do not want to recreate the container, see #477 (in short - you have to inspect the container to see its environment variables).
